ZUGFeRD/InvoiceDescriptor.cs

```diff
@@ -302,7 +300,7 @@ public class InvoiceDescriptor
         /// <summary>
         /// Detailed information about payment terms               
         /// </summary>
-        public PaymentTerms PaymentTerms { get; set; }
+        public List<PaymentTerms> PaymentTerms { get; set; } = new List<PaymentTerms>();
```

Comment:
should we use this change to make the list private like with some of the other lists?

Reply:
Preferably, yes.
It's already kind of a breaking change, and it might be a good opportunity to make the property private.
I didn't want to make that decision, and used same access modifier as the other properties of type List.

Reply:
If you have the time, please go ahead and make this particular one private. For the rest, I'll create an issue for the next major release.

Reply:
This is going to break a lot of the existing tests because the property is accessed directly.
Going to update / rewrite them, too.

Reply:
The access modifier of the property has been changed to `private` and any external access has been amended to either use `GetTradePaymentTerms()` or `AddTradePaymentTerms()`.

Silly me, I forget that the List can still be cleared this way. So, no harm observed here, all tests are green.